src/Panel/Home.php

```diff
@@ -239,10 +238,19 @@ public function url(): string
 
 		// Check if the user can access the URL
 		if ($this->hasAccess($path) === true) {
-			return Panel::url($path);
+			return $this->panel->url($path);
 		}
 
 		// Try to find an alternative
 		return $this->alternative();
 	}
+
+	/**
+	 * Returns the current user
+	 * @since 6.0.0
+	 */
+	public function user(): User|null
+	{
+		return $this->kirby->user();
+	}
```

Comment:
Wouldn't it be enough to get that once in the methods above and store it in a var instead of adding another method?

Reply:
I had this in the constructor, but the other can change during a request (e.g. impersonation) and then the troubles start (quite common in our tests) as the instance is also cached in the Panel class and the Panel instance is cached in the App class.